Sources/Extensions/Foundation/CalendarExtensions.swift

```diff
@@ -20,4 +20,23 @@ public extension Calendar {
     public func numberOfDaysInMonth(for date: Date) -> Int {
         return range(of: .day, in: .month, for: date)?.count ?? 0
     }
+    
+    /// SwifterSwift: Return the number of days in the year for specified 'Date'.
+    ///
+    /// - Parameter date: the date from which the year is obtained
+    /// - Returns: number of days in the year of 'date'
+    public func numberOfDaysInYear(for date: Date) -> Int {
+        return range(of: .day, in: .year, for: date)?.count ?? 0
+    }
+    
+    /// SwifterSwift: Return the number of months in a year for specified 'date'
+    ///
+    ///         let calendar = Calendar(identifier: .gregorian)
+    ///         calendar.numberOfMonthsInYear(for: Date()) -> 12
+    ///
+    /// - Parameter date: the date from which the year is obtained
+    /// - Returns: number of days in the year of 'date'
+    public func numberOfMonthsInYear(for date: Date) -> Int {
```

Comment:
Are there other calendars with more months than 12? I'm not sure if this is necessary